content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -245,6 +243,9 @@ different Kubernetes components.
 | `DryRun` | `true` | GA | 1.19 | - |
 | `DynamicAuditing` | `false` | Alpha | 1.13 | 1.18 |
 | `DynamicAuditing` | - | Deprecated | 1.19 | - |
+| `DynamicKubeletConfig` | `false` | Alpha | 1.4 | 1.10 |
+| `DynamicKubeletConfig` | `true` | Beta | 1.11 | 1.22 |
+| `DynamicKubeletConfig` | `false` | Deprecated | 1.22 | |
```

Comment:
/hold

to confirm the decision to switch if off by default (here: https://github.com/kubernetes/kubernetes/pull/102966)